bundles/core/org.openhab.core.jsr223/META-INF/MANIFEST.MF

```diff
@@ -6,8 +6,12 @@ Bundle-Vendor: openHAB.org
 Bundle-Version: 1.8.0.qualifier
 Bundle-SymbolicName: org.openhab.core.jsr223;singleton:=true
 Bundle-ActivationPolicy: lazy
-Require-Bundle: org.openhab.model.item
+Require-Bundle: org.openhab.model.item,
+ org.openhab.core.persistence,
+ org.openhab.model.script
```

Comment:
please avoid using require-bundle deps in favour of import-package statements (see also https://github.com/openhab/openhab/wiki/How-To-Contribute#OSGi Bundles)